released/packages/coq-ltac-iter/coq-ltac-iter.1.1.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/gmalecha/coq-ltac-iter"
+dev-repo: "git+https://github.com/gmalecha/coq-ltac-iter.git"
+bug-reports: "https://github.com/gmalecha/coq-ltac-iter/issues"
+authors: ["Gregory Malecha"]
+license: "BSD"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.9" & < "8.10~"}
+]
+synopsis:
+  "Access hint databases from tactics"
+description: "Provides tactics for performing various forms of iteration over hypotheses, hint databases, and other collections."
+
+url {
+  src: "https://github.com/gmalecha/coq-ltac-iter/archive/1.1.1.tar.gz"
```

Comment:
```suggestion
  src: "https://github.com/gmalecha/coq-ltac-iter/archive/v1.1.1.tar.gz"
```

Reply:
I think when you force-pushed, you undid this change?